libraries/azure_event_hub_event_hub.rb

```diff
@@ -0,0 +1,44 @@
+require 'azure_generic_resource'
+
+class AzureEventHubEventHub < AzureGenericResource
+  name 'azure_event_hub_event_hub'
+  desc 'Verifies settings for Event Hub Event Hub'
```

Comment:
`Event Hub` is repeated twice here and in the desc for the backward compatible resource